models/wholeBody_ct_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,182 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.1.0",
+    "changelog": {
+        "0.1.0": "complete the model package",
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "1.1.0",
+    "pytorch_version": "1.13.0",
+    "numpy_version": "1.21.2",
+    "optional_packages_version": {
+        "nibabel": "4.0.1",
+        "pytorch-ignite": "0.4.9"
+    },
```

Comment:
Hi @tangy5 , according to the latest change related to #189 , could you please also add a "name" tag here?
the name here will be the display name in the webpage

Reply:
Thanks, added.